### ISSUE TITLE
History of heat kernel estimates

### ISSUE THREAD

**Issue by pabryan** [pabryan/rough_metrics_heat_kernel_regularity#1]
This is of course, huge but we should give some highlights particularly in our context and for the statements about existence and regularity, RCD spaces, minimal heat kernels, Varadahn's formula and Harnack inequalities.

**Comment by lashputin** [comment]
Yes, maybe we need to make a list of things that has motivated this work. In particular, to say that in the smooth setting, or more generally in the setting when we have a  metric structure, Varadhan's formula gives a relationship of the distance to small time asymptotics of the Dirichlet heat kernel, and that in our work, we have freedom to choose distances, but we build the heat kernel to restrict the kinds of reasonable distances we can expect.
